app/src/androidTest/java/org/mozilla/focus/activity/screenshots/SettingsScreenshots.java

```diff
@@ -77,50 +77,54 @@ public void clearSettings() {
     public void takeScreenShotsOfSettings() throws Exception {
         openSettings();
 
-        assertTrue(TestHelper.settingsHeading.waitForExists(waitingTime));
         Screengrab.screenshot("Settings_View_Top");
 
         /* Language List (First page only */
-        UiObject LanguageSelection = TestHelper.settingsList.getChild(new UiSelector()
-                .className("android.widget.LinearLayout")
-                .instance(0));
-        LanguageSelection.click();
-        TestHelper.settingsHeading.waitUntilGone(waitingTime);
-
-        UiObject CancelBtn =  device.findObject(new UiSelector()
-                .resourceId("android:id/button2")
-                .enabled(true));
+        onView(withText(R.string.preference_language))
```

Comment:
@liuche oh no, ids are way better than strings for sure. For example, if we use strings, it will fail if the localization is not properly done. For this PR, I switched some code back to espresso, but reverted this one back to using IDs.  The reason is that Espresso is way faster than Uiautomator, and running tests on 70 locals, time adds up quickly.  Espresso does have ids too - R.id.* , but I couldn't find the one for the cancel button. And it was failing on locals without 'cancel' word translated - so for this I'm forced to use uiautomator call with IDs.